LoRaEngine/LoraKeysManagerFacade/SearchDeviceByDevEUI.cs

```diff
@@ -43,29 +45,30 @@ public async Task<IActionResult> GetDeviceByDevEUI([HttpTrigger(AuthorizationLev
 
         private async Task<IActionResult> RunGetDeviceByDevEUI(HttpRequest req, ILogger log)
         {
-            var devEUI = req.Query["DevEUI"];
-            if (StringValues.IsNullOrEmpty(devEUI))
+            var rawDevEui = req.Query["DevEUI"];
+            if (StringValues.IsNullOrEmpty(rawDevEui))
```

Comment:
why not start using TryParse on the Eui for all the parameter conversions? We could also get rid of the EUIValidator, no?

Reply:
We cannot get rid of the `EuiValidator` entirely since it also protects against `default`, but I refactored to get rid of the duplication in 2fdccb71d6985d01c8d81c1a696958524477f81f